docs/configuration.md

```diff
@@ -5,7 +5,7 @@ Additional configuration files might be needed by some modules
 depending on their needs and if they are used in the pipeline at all.
 
 #### ENV variables
-Every configuration option in the base YAML file can be overridden by using ENV variable.
+Every configuration option in the base YAML file can be overridden(the configuration MUST be present in the yaml config for the ENV to be loaded, it can be even empty string) by using ENV variable.
```

Comment:
```suggestion
Every configuration option in the base YAML file can be overridden (the configuration MUST be present in the yaml config for the ENV to be loaded, it can be even empty string) by using ENV variable.
```